README.md

```diff
@@ -5,6 +5,9 @@ sqlite modern cpp wrapper
 
 This library is a lightweight modern wrapper around sqlite C api .
 
+**Heads up!**
+The current master contains a _breaking change_ compared to the 2.4 release for database names with non-ASCII characters. For details, see [issue 82](https://github.com/aminroosta/sqlite_modern_cpp/issues/82).
+
```

Comment:
@zauguin Could you please add a **Breaking Changes** section at the end of REAME, i will update previous breaking changes and we will add new changes to that section :-)